WooCommerce/Classes/POS/Analytics/POSCollectOrderPaymentAnalytics.swift

```diff
@@ -83,6 +96,11 @@ final class POSCollectOrderPaymentAnalytics: CollectOrderPaymentAnalyticsTrackin
     func resetCheckoutTapCountTracker() {
         checkoutTapCount = 0
     }
+
+    private func trackElapsedTimeFromOrderCreationToCardReady() {
+        let elapsedTime = cardReaderReady - orderCreated
```

Comment:
Let's track `orderCreated` both when `newOrder` is created and the existing one is reused to align with Android p1739888487411149/1739867347.923049-slack-C070SJRA8DP.

Reply:
Thanks! Updated here: https://github.com/woocommerce/woocommerce-ios/pull/15151/commits/1bafc533e9328480aef9d1c16dfb08c28f0e5ecc and confirmed that we only track it on `newOrder` and `orderUpdated` cases.

Reply:
Thanks! 🚀